pony/README.md

```diff
@@ -0,0 +1,7 @@
+# Pony
+
+## How to
+
+* Install [ponyc](https://github.com/ponylang/ponyc)
+* Run `ponyc`
```

Comment:
This step fails for me with

```
Building builtin -> /usr/lib/pony/0.28.0/packages/builtin
Building . -> /home/axel/CompetenceGroups/AQE/hello-world-in-90-languages/pony
Error:
http: couldn't locate this path
Error:
/home/axel/CompetenceGroups/AQE/hello-world-in-90-languages/pony/hello-world.pony:1:1: can't load package 'http'
use "http"
^
```

Reply:
Yeah, woops. ponyc can't handle dependencies - i'll update the README to recommend https://github.com/ponylang/pony-stable

Reply:
There's no date header yet... woops